dashboard/app/jobs/aichat_request_chat_completion_job.rb

```diff
@@ -50,6 +50,14 @@ def perform(request:, locale:)
     request.update!(response: response, execution_status: status)
   end
 
+  private def multimodal?(request)
+    messages = [*request.stored_messages, request.new_message]
+    messages_with_assets_count = messages.count do |message|
+      message['assets']&.any?
+    end
+    return messages_with_assets_count>0
```

Comment:
nit: you can simplify the return value to just

```
messages.count do |message|
  message['assets']&.any?
end
```

rather than storing a `message_with_assets_count` local variable